components/search_results/files_filter_menu.tsx

```diff
@@ -16,13 +19,32 @@ type Props = {
 };
 
 export default function FilesFilterMenu(props: Props): JSX.Element {
+    const toolTip = (
+        <Tooltip
+            id='files-filter-tooltip'
+            className='hidden-xs'
+        >
+            <FormattedMessage
+                id='channel_info_rhs.menu.files.filter'
+                defaultMessage='Filter'
+            />
+        </Tooltip>
+    );
     return (
         <div className='FilesFilterMenu'>
             <MenuWrapper>
-                <span className='action-icon dots-icon'>
-                    {props.selectedFilter !== 'all' && <i className='icon-dot'/>}
-                    <i className='icon icon-filter-variant'/>
-                </span>
+                <OverlayTrigger
+                    className='hidden-xs'
+                    delayShow={500}
+                    placement='top'
+                    overlay={toolTip}
+                    rootClose={true}
+                >
+                    <span className='action-icon dots-icon'>
+                        {props.selectedFilter !== 'all' && <i className='icon-dot'/>}
+                        <i className='icon icon-filter-variant'/>
```

Comment:
Could you please use an icon from `compass-icons` like here 
https://github.com/mattermost/mattermost-webapp/blob/9032cc8570dc3c22842c5550971063fdeb30e9d2/components/advanced_create_post/advanced_create_post.tsx#L1549
Icon you will be needing could be found here https://mattermost.github.io/compass-icons/
<img width="162" alt="Screenshot 2022-11-07 at 7 55 57 PM" src="https://user-images.githubusercontent.com/16203333/200334322-51fd8851-a7f4-48a6-a698-2c5af2851773.png">

Reply:
Added the requested changes, could please review